### ISSUE TITLE
[Terminal Preview - Tab Color] : Screen reader is not announcing the already selected color when navigating using keyboard.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15980]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

**Test Environment:** 
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Set shortcut for 'Set tab color'.
4. Now navigate back to the Window terminal page and Open color pop-up using shortcut.
5. Navigate over the color using Narrator and observe the issue.

**User Experience: **
When a screen reader does not provide complete information, it is difficult for people using assistive technology (like screen readers) to understand the functionality of the control. Having complete information about the control will help users to use the control as per their required task. 
 
**MAS Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Attachment :

### Expected Behavior

Screen reader should announce the already selected color information when navigating using keyboard.

### Actual Behavior

Screen reader is not announcing the already selected color when navigating using keyboard.
